pkg/urbit/Makefile

```diff
@@ -32,9 +32,9 @@ CFLAGS := $(CFLAGS)
 all: urbit urbit-worker hashtable_tests jam_tests mug_tests
```

Comment:
The "all_exes" variable (above) also needs to be prefixed with the build directory.